pkg/commands/game/lookForPlayers.go

```diff
@@ -0,0 +1,415 @@
+package game
+
+import (
+	"errors"
+	"fmt"
+	"github.com/bwmarrin/discordgo"
+	"github.com/itfactory-tm/thomas-bot/pkg/command"
+	"github.com/itfactory-tm/thomas-bot/pkg/db"
+	"github.com/itfactory-tm/thomas-bot/pkg/embed"
+	"log"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+)
+
+//TODO: make configurable in config file
+//LFPDesk channel id
+const lfpDeskID = "828205635249635369"
+//LFP Request channel id
+const lfpReqID = "828204894187421696"
+
+// LookCommand contains the /lookforplayers command
+type LookCommand struct{
+	db           db.Database
+}
+
+// NewSearchCommand gives a new SearchCommand
+func NewLookCommand(dbConn db.Database) *LookCommand {
+	return &LookCommand{
+		db:           dbConn,
+	}
+}
+
+// Register registers the handlers
+func (l *LookCommand) Register(registry command.Registry, server command.Server) {
+	registry.RegisterInteractionCreate("lookforplayers",l.SearchCommand)
+	registry.RegisterMessageReactionAddHandler(l.handleReactionAdd)
+	registry.RegisterMessageReactionRemoveHandler(l.handleReactionRemove)
+}
+
+//InstallSlashCommands registers the slash commands
+func (l *LookCommand) InstallSlashCommands(s *discordgo.Session) error {
+	_, err := s.ApplicationCommandCreate("", "773847927910432789", &discordgo.ApplicationCommand{
+		Name:        "lookForPlayers",
+		Description: "Send out an invitation to look for players!",
+		Options: []*discordgo.ApplicationCommandOption{
+					{
+						Type:        discordgo.ApplicationCommandOptionString,
+						Name:        "game",
+						Description: "Name of the game",
+						Required:    true,
+					},{
+						Type:        discordgo.ApplicationCommandOptionInteger,
+						Name:        "amount",
+						Description: "Amount of people you need for the game",
+						Required:    true,
+					},{
+						Type:        discordgo.ApplicationCommandOptionRole,
+						Name:        "notifyrole",
+						Description: "Notify a role with your invitation!",
+						Required:    false,
+					},{
+						Type:        discordgo.ApplicationCommandOptionString,
+						Name:        "time",
+						Description: "At what time do you want to play? Format hh:mm (example: 15:45)",
+						Required:    false,
+					},
+				},
+	})
+	return err
+}
+
+func (l *LookCommand) SearchCommand(s *discordgo.Session, i *discordgo.InteractionCreate) {
+	var name, selectedRoleID string
```

Comment:
can be moved down, saves allocating variables in case of `if i.ChannelID != lfpReqID{`